RELEASE.md

```diff
@@ -20,25 +20,30 @@ Further details about versioning can be found in the [semver 2.0.0 specification
 
 ## Release Process
 
-### Manual Steps (needs to be done per package)
+### Manual Steps For Alpha Release
 
 1. `git checkout 4.x`: Verify you are on the `4.x` base branch
-2. `git checkout -b release/package-name/bumped-version`: Checkout a branch with the `package-name` and `bumped-version` e.g. `git checkout -b release/web3-packagetemplate/1.0.1`
+2. `git checkout -b release/bumped-version`: Create and checkout a branch with the `bumped-version` e.g. `git checkout -b release/4.0.0-alpha.0`
+    - `bumped-version` of release branch should be of main web3 package.
```

Comment:
I'm not sure what you mean by this

Reply:
- `bumped-version` of release branch should be of main web3 package. 

is for release branch naming as we will have independent versioning so initially we can name release branch same like our main web3 package.